src/models/thelio-mira-r1.0/README.md

```diff
@@ -0,0 +1,81 @@
+# Thelio Mira (thelio-mira-r1.0)
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Thelio Mira](./img/mira-front.jpg)
```

Comment:
Can we use this instead?

https://assets.system76.com/products/thelio-mira-r1/feature.png?fit=cover&width=500